docs/self-managed/modeler/web-modeler/troubleshooting/troubleshoot-other-problems.md

```diff
@@ -0,0 +1,26 @@
+---
+id: troubleshoot-other-problems
+title: "Troubleshoot other problems"
+sidebar_label: "Other problems"
+description: "Troubleshooting guide for other issues in Web Modeler."
```

Comment:
💭 This guide didn't really fit into the other existing WM troubleshooting sections - at the other hand, I didn't want to add just another specific page for one problem which is why I ended up with an "other issues" page.

@camunda/tech-writers feel free to adjust this - I'm also fine with introducing a dedicated page.